includes/oauth/class-oauth-transients.php

```diff
@@ -165,9 +166,13 @@ public static function delete( $id, $scope ) {
 	 * @return mixed The value if it was set, false otherwise.
 	 */
 	public static function set( $id, $scope, $value ) {
+		if ( empty( $id ) ) {
+			return false;
```

Comment:
nit: but let's add some logging in this case so we can surface this error?

Maybe add the check and logging in the `generate_csrf_token` method instead

Reply:
Good call, I'll keep both, since `set()` is a public method.

Reply:
Added in 5670bfbb2f1ad0391b1625f5a96b8653cf977b1d